bindings/python/FloatingBaseEstimators/src/FloatingBaseEstimators.cpp

```diff
@@ -168,7 +172,8 @@ void CreateFloatingBaseEstimator(pybind11::module& module)
     py::class_<Source<Output>>(module, "FloatingBaseEstimatorOutputSource");
 
     py::class_<FloatingBaseEstimator, Source<Output>> floatingBaseEstimator(module,
-                                                                            "FloatingBaseEstimator");
+                                                                            "FloatingBaseEstimato"
+                                                                            "r");
```

Comment:
Something strange happened here. You can force newline with `floatingBaseEstimator( //` if you need more control.